src/Interface/STS.agda

```diff
@@ -39,7 +39,7 @@ private
 
 -- small-step to big-step transformer
 
-module _ (_⊢_⇀⟦_⟧ᵇ_ : C → S → ⊤ → S → Type) (_⊢_⇀⟦_⟧_ : C → S → Sig → S → Type) where
+module _ {_⊢_⇀⟦_⟧ᵇ_ : C → S → ⊤ → S → Type} {_⊢_⇀⟦_⟧_ : C → S → Sig → S → Type} where
```

Comment:
If these parameters are explicit, then the underscores are not filled in properly by Agda in the *Agda* status buffer and it makes statements of goals, errors, etc. hard to comprehend.  When the parameters are implicit, the underscores work as desired.  Specifying the parameters now requires curly braces, but I think that's a small price to pay for making the underscores work as expected.